care/facility/models/patient.py

```diff
@@ -806,13 +807,25 @@ class PatientNotes(FacilityBaseModel, ConsultationRelatedPermissionMixin):
         related_name="replies",
     )
     note = models.TextField(default="", blank=True)
+    parent_note = models.ForeignKey(
```

Comment:
`root_note` would be more appropriate term right? `parent_note` might be confused with being only for immediate parent only (and not for potential grand parents)

```suggestion
    root_note = models.ForeignKey(
```

Reply:
got it